be/src/runtime/runtime_filter_worker.cpp

```diff
@@ -773,23 +798,34 @@ void RuntimeFilterWorker::_deliver_broadcast_runtime_filter_local(PTransmitRunti
         finst_id->set_lo(id.lo);
     }
     _exec_env->add_rf_event({param.query_id(), param.filter_id(), "", "DELIVER_BROADCAST_RF_LOCAL"});
-    _receive_total_runtime_filter(param, nullptr);
+    _receive_total_runtime_filter(param);
+}
+
+void RuntimeFilterWorker::_deliver_part_runtime_filter(std::vector<TNetworkAddress>&& transmit_addrs,
+                                                       PTransmitRuntimeFilterParams&& params, int transmit_timeout_ms) {
```

Comment:
It seems no need to use rvalue-ref, I think const ref is enough.
It will make me difficult to understand if you are using rvalue-ref, because I will consider you are going to change the input value. But in this case, you don't change the input arguments.